### PR TITLE
Theme correction

### PR DESCRIPTION
**Pull request by MeyAyre**
Changed the UI to adhere to a coherent theme across the app.

### CODE REVIEW

**Review by jc-hayes — APPROVED** [review]

**Review by Juanitaag1 — APPROVED** [review]

**Comment by Juanitaag1** [comment]
I checked out the branch and it would not run. Xcode message:The document "Main.storyboard" could not be opened. Multiple elements have the ID "cPa-gy-q4n" . Anyone getting it to run?

File: GYBITG/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -92,68 +92,104 @@
...
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="centerX" secondItem="i4i-eX-Fe0" secondAttribute="centerX" id="msN-G7-sOe"/>
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="centerX" secondItem="OKK-Yi-t2O" secondAttribute="centerX" id="tI6-dZ-dF6"/>
+                            <constraint firstItem="OKK-Yi-t2O" firstAttribute="leading" secondItem="ohI-EW-ROe" secondAttribute="leading" id="uWZ-ji-tJd"/>
+                            <constraint firstItem="uLi-pb-8FA" firstAttribute="leading" secondItem="O1u-W8-tvY" secondAttribute="leading" constant="79" id="uZl-9G-pmo"/>
+                            <constraint firstItem="OKK-Yi-t2O" firstAttribute="top" secondItem="rGB-cO-yXL" secondAttribute="bottom" constant="35" id="wHZ-DZ-pca"/>
+                        </constraints>
                         <viewLayoutGuide key="safeArea" id="O1u-W8-tvY"/>
                     </view>
-                    <tabBarItem key="tabBarItem" title="" image="icons-female-profile-50" id="cPa-gy-q4n"/>
```
Comment (Juanitaag1) [inline GYBITG/Base.lproj/Main.storyboard:152]: I think it was this line that caused the error. Try changing the id to get a different id. This should make the error of more than 1 id are same go away.

**Review by Juanitaag1 — CHANGES_REQUESTED** [review]

File: GYBITG/Base.lproj/Main.storyboard (hunk 1 above)
Comment (jc-hayes) [reply GYBITG/Base.lproj/Main.storyboard:152]: I commented out line 611, in Main.Storyboard: `<tabBarItem key="tabBarItem" title="Account" image="second" id="cPa-gy-q4n"/>` and the app built. 

Line 152 has the reference to the profile icon, so we want to keep that one. I think it's line 611 that should be removed.

Comment (MeyAyre) [reply GYBITG/Base.lproj/Main.storyboard:152]: I just commented out line 611 and it's building now. I pushed that change

**Review by Juanitaag1 — APPROVED** [review]

**Review by shariefyoussef — APPROVED** [review]

**Pull request merged**